pandas/tests/frame/test_analytics.py

```diff
@@ -658,21 +658,21 @@ def test_numeric_only_flag(self, meth):
         pytest.raises(TypeError, lambda: getattr(df2, meth)(
             axis=1, numeric_only=False))
 
-    def test_mixed_ops(self):
+    @pytest.mark.parametrize('op', ['mean', 'std', 'var',
```

Comment:
do we have these as fixtures?

Reply:
Not AFAICT

Reply:
cython_table_items yields these (well it yields them all, maybe need to make a subset)

Reply:
> well it yields them all, maybe need to make a subset

Consider:

```
@pytest.mark.parametrize('op', SPECIFIC_EXPLICIT_LIST)
def test_thing(self, op):

def test_thing(self, cython_table_items):
     if cython_table_items not in SPECIFIC_EXPLICIT_LIST:
          return
```

There are good use cases for fixtures, this isn't near the top of that list.